documentation/userguide/docs/metadata_forms.md

```diff
@@ -0,0 +1,80 @@
+# **Metadata Forms**
+
+**Concepts**
+
+Metadata forms allow users to add structured contextual information to various entities in the Data.all platform. By creating and attaching metadata forms, user can standardize and enrich metadata in a customizable way.
+
+Metadata forms serve several key purposes:
+
+- Improve data discovery by enabling more consistent, complete, and meaningful metadata
+- Capture domain-specific or organizational metadata standards
+- Enforce governance policies around data documentation
+- Facilitate data lineage and impact analysis
+- Streamline metadata management workflows
```

Comment:
I am a bit confused with `Enforce governance policies around data documentation` are we talking about enforcing metadata standards ensuring completeness? And if so, let's include other entities not only data.

Also on data lineage... I would not name data lineage as it can lead to misunderstandings of the feature (e.g. people might assume this provides a dependency graph of resources).